src/test-run/index.js

```diff
@@ -34,12 +53,13 @@ export default class TestRun extends Session {
 
     async _start () {
         this.started = true;
+        this.emit('start');
 
         try {
             await this.test.fn(this);
         }
         catch (err) {
-            this.errs.push(err);
+            this.errs.push(new TestRunErrorFormattableAdapter(err, this.browserConnection.userAgent, '', this.failedCommandCallsite));
```

Comment:
Screenshots are not implemented. Where can we get `screenshotPath` without this?